File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT2_JAVA/src/Organisms/Animals/Wolf.java
package Organisms.Animals;

import Game.World;

import java.awt.*;
import java.io.Serializable;

public class Wolf extends Animal implements Serializable {
    public Wolf(World world,int x, int y) {
        super("W", "Wolf", Color.GRAY, 9, 5, 1, world,x,y);
    }
    @Override
    public Wolf getChild(int x, int y)
    {
        return new Wolf(world,x,y);
    }
}

File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT3_PYTHON/Organisms/Animals/wolf.py
from Organisms.Animals.animal import Animal


class Wolf(Animal):
    def __init__(self, position_x, position_y, world):
        super().__init__(position_x, position_y, "Wolf", 'W', 9, 5, world, 1, (100, 100, 100))

    def getchild(self, newposition):
        return Wolf(newposition[0], newposition[1], self.world)
